src/Admin/AdminInterface.php

```diff
@@ -148,7 +148,7 @@ public function setParentFieldDescription(FieldDescriptionInterface $parentField
     /**
      * Get parent field description.
      *
-     * @return FieldDescriptionInterface The parent field description
+     * @return FieldDescriptionInterface|null The parent field description
```

Comment:
IMHO you can remove the comment as it is self-explaining

Reply:
Done